src/test/java/io/managed/services/test/rhoas/KafkaRhoasBasicTests.java

```diff
@@ -62,6 +63,7 @@ public class KafkaRhoasBasicTests extends TestBase {
 
     // TODO enterprise: change logic of name assignment back once we have quota for more than 1 enterprise kafka instance
     private static final String KAFKA_INSTANCE_NAME = Environment.IS_ENTERPRISE ? "enterprise-test" : "e2e-cli-test-" + Environment.LAUNCH_SUFFIX;
+    //private static final String CLI_CREATE_UPDATE_KAFKA_INSTANCE_NAME = Environment.IS_ENTERPRISE ? "enterprise-test-kfk" : "e2e-test-kfk-" + Environment.LAUNCH_SUFFIX;
```

Comment:
should this commented line be removed? or should it be uncommented?

Reply:
Fixed in [abe6962](https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/pull/527/commits/abe69623b209277eced5d1b9a90940df0a755868)